packages/apps/plugins/plugin-navtree/src/NavTreePlugin.tsx

```diff
@@ -43,14 +44,23 @@ export type NavTreePluginProvides = SurfaceProvides &
   GraphBuilderProvides &
   TranslationsProvides;
 
+type NavTreeState = { root?: NavTreeItemGraphNode };
+
+type NavTreeSettings = { openItemIds?: OpenItemIds };
+
 export const NavTreePlugin = (): PluginDefinition<NavTreePluginProvides> => {
-  const state = create<{ root?: NavTreeItemGraphNode; openItemIds?: OpenItemIds }>({});
+  const state = create<NavTreeState>({});
+
+  const settings = new LocalStorageStore<NavTreeSettings>('dxos.org/settings/navtree', {
```

Comment:
A couple things on this:
- these aren't settings this is just state still
- local storage store doesn't automatically map everything to local storage so you can include the root here too and just not map it to local storage